contents/blog/leo-anthias-interview.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2021-07-28
+title: Towards Open Source Data Science - An interview with Datapane CEO, Leo Anthias
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/leo-anthias-interview.png
+---
+
+Unlike most tech founders, Leo Anthias didn’t study computer science at university. Instead, he took Persian History and Modern American Drama after a chance encounter with a professor who made medieval history fascinating. Despite his non-tech academic background, however, Leo has been coding for years, from early roles writing Perl scripts for freelance clients to his current job as CEO and co-founder of Datapane.
+
+A former YC alumnus, his fondest memory from that time is the absolute focus and discipline that was needed to perfect one’s product strategy, ideal customer profile, and marketing and distribution. His strongest productivity tip to date is to focus on one thing at a time, and this pairs well with his ability to perform well under pressure - especially when running against externally imposed client deadlines.
+
+> It's really nice to just focus on one thing. For the last two or three days, I've really only focused on our onboarding flows for guides and tutorials for new users. And being able to just focus on that one thing has meant that probably I've gotten a lot more done than if I tried to do that while at the same time running a hiring process, while at the same time having a fundraising process, whatever. Even though all of those things are of equal importance, it's really key to just pick one of them at a time.
+
+After returning to the UK, Leo co-founded another company: a data science model deployment platform. Their goal was to make it easy for engineers to publish a model to the cloud, scale it, and integrate it. Part of that service was a library they built for their customers, which they eventually spun out into a separate product - Datapane. 
+
+Datapane’s birth was inspired by data scientists leaning more on no-code tools and plugins for analysis, visualization, and collaboration. Being locked into proprietary software ecosystems makes it hard to publish new research with the world, so Datapane aims to simplify going from coding data in Python to sharing snippets on Medium or in PowerPoint decks. Leo and his team thus optimize for the number of people building and sharing reports.
+
+Datapane uses Django as a base, Alpine JS for the frontend, Tailwind CSS for the design, React for the interactive slides, and Kubernetes for handling orchestration, workload execution, and other cloud-based tasks. One of Datapane’s best features is being able to create reports in your browser (using a Markdown editor) without needing to go through Python libraries. Leo and his team are also looking to build a mobile app for report viewing and sharing in the near future.
```

Comment:
We could have one or two images of the Datapane application showing these things. Something Leo could provide?

As a geek, I'd be interested in seeing what this looks like.

May break up the text a bit too?

Reply:
here ya go

![image](https://user-images.githubusercontent.com/3541695/127021097-99ff4a95-5fee-4a22-8856-7171446e71ef.png)
![image](https://user-images.githubusercontent.com/3541695/127020720-663dfd87-5665-4c4b-a49c-7f1519df93d0.png)
![image](https://user-images.githubusercontent.com/3541695/127020981-44d8310a-e78a-4801-897f-a8dc5ac6b1ef.png)

![image](https://user-images.githubusercontent.com/3541695/127020810-5a87a883-7212-4e25-9ce8-25dfbf8d64a8.png)

https://www.loom.com/share/d86db0cc1bda4d16bd0f63342c77243f